sdk/tables/azure-data-tables/azure/data/tables/_table_service_client.py

```diff
@@ -156,6 +156,30 @@ def create_table(
         table.create_table(**kwargs)
         return table
 
+    @distributed_trace
```

Comment:
I like having it on the `TableServiceClient`, it doesn't make as much sense to me having it on `TableClient` since that's what you're trying to create